llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -2690,6 +2690,44 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
     SC.Done(&I);
   }
 
+ // Perform a bitwise OR on the horizontal pairs (or other specified grouping)
```

Comment:
If I only read this description, I'd have no idea what this does

Reply:
Done